chapter_preliminaries/probability.md

```diff
@@ -354,54 +808,235 @@ $$P(H = 1| D_1 = 1, D_2 = 1)
+No matter how long we watch someone tossing a fair coin, 
+we will never be more or less than 50% certain 
+that the next toss will come up heads.
+These terms owe to literature in mechanical modeling,
+(see e.g., :cite:`der2009aleatory` for a review on this aspect of [Uncertainty Quantification](https://en.wikipedia.org/wiki/Uncertainty_quantification)).
+It's worth noting that these terms constitute a slight abuse of language.
+The term *epistemic* refers to anything concerning *knowledge*
+and thus in the philosophical sense, all uncertainty is epistemic.
+
+
+We saw that sampling data from some unknown probability distribution 
+can provide us with information that can be used to estimate
+the parameters of the data generating distribution.
+That said, the rate at which this is possible can be quite slow. 
+In our coin tossing example (and many others) 
+we can do no better than to design estimators
+that converge at a rate of $1/\sqrt{n}$. 
+This means that by going from 10 to 1000 observations (usually a very achievable task) 
+we see a tenfold reduction of uncertainty, 
+whereas the next 1000 observations help comparatively little, 
+offering only a 1.41 times reduction. 
+This is a persistent feature of machine learning: 
+while there are often easy gains, it takes a very large amount of data, 
+and often with it an enormous amount of computation to make even further gains. 
+For an empirical review of this fact for large scale language models see :cite:`revels2016forward`. 
+
+We also sharpened our language and tools for statistical modeling. 
+In the process of that we learned about conditional probabilities 
+and about one of the most important equations in statistics---Bayes' theorem. 
+It is an effective tool for decoupling information conveyed by data 
+through a likelihood term $P(B|A)$ that addresses 
+how well observations $B$ match a choice of parameters $A$,
+and a prior probability $P(A)$ which governs how plausible 
+a particular choice of $A$ was in the first place.
+In particular, we saw how this rule can be applied
+to assign probabilities to diagnoses,
+based on the efficacy of the test *and* 
+the prevalence of the disease itself (i.e., our prior).
+
+Lastly, we introduced a first set of nontrivial questions 
+about the effect of a specific probability distribution,
+namely expectations and variances. 
+While there are many more than just linear and quadratic 
+expectations for a probability distribution, 
+these two already provide a good deal of knowledge 
+about the possible behavior of the distribution. 
+For instance, [Chebyshev's Inequality](https://en.wikipedia.org/wiki/Chebyshev%27s_inequality)
+states that $P(|X - \mu| \geq k \sigma) \leq 1/k^2$, 
+where $\mu$ is the expectation, $\sigma^2$ is the variance of the distribution 
+and $k > 1$ is a confidence parameter of our choosing. 
+It tells us that draws from a distribution lie 
+with at least 50% probability within a $2 \sigma$ interval of the expectation. 
```

Comment:
$2 \sigma$ -> $2\sqrt{2} \sigma$

Reply:
Edited as:

It tells us that draws from a distribution lie 
with at least 50% probability 
within a $[-\sqrt{2} \sigma, \sqrt{2} \sigma]$
interval centered on the expectation.